docs/03-tutorials/00-api-exposure/apix-03-expose-multiple-services.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Expose multiple services on the same host
+---
+
+This tutorial shows how to expose a workload with multiple services that share the same host on different paths.
```

Comment:
The tutorial has two subsections, "Multiple services definition on different paths" and "Root level service definition and multiple services definition on different paths".
What's their relationship? Are they alternatives to each other (my guess), or does one follow after the other? Or is the second one a specialized case of the first one? 

We should make their relationship clear in this introductory section.

Reply:
Then I'd add that information. I shamelessly stole from your explanation:

```suggestion
This tutorial shows how to expose a workload with multiple services that share the same host on different paths.

You can either define a service at the root level, which is applied to all paths except the ones you've explicitly set service for  at the rules level, or you can just define different services on each path separately without the need to define a root service.
```